nilearn/plotting/matrix_plotting.py

```diff
@@ -338,19 +338,14 @@ def plot_contrast_matrix(
 
     Parameters
     ----------
-    contrast_def : :obj:`str` or :class:`numpy.ndarray` of shape (n_col),\
-    or :obj:`list` of :obj:`str`, or :class:`numpy.ndarray` of shape (n_col)
-
-        where ``n_col`` is the number of columns of the design matrix, (one
-        array per run).
-        If only one array is provided when there are several runs,
-        it will be assumed that the same :term:`contrast` is desired
-        for all runs. The string can be a formula compatible with
-        :meth:`pandas.DataFrame.eval`. Basically one can use the name of the
-        conditions as they appear in the design matrix of the fitted model
-        combined with operators +- and combined with numbers with operators
-        +-`*`/.
-
+    contrast_def : :obj:`str` or :class:`numpy.ndarray` of shape[1] <= n_col \
+        where ``n_col`` is the number of columns of the design matrix.
+        The string can be a formula compatible
+        with :meth:`pandas.DataFrame.eval`.
+        Basically one can use the name of the conditions
+        as they appear in the design matrix of the fitted model
+        combined with operators +-
+        and combined with numbers with operators +-`*`/.
```

Comment:
as far as I can tell the code in this function cannot handle list of strings or list of arrays as inputs